ptsprojects/mynewt/mesh.py

```diff
@@ -40,6 +40,10 @@
 from binascii import hexlify
 import random
 from time import sleep
+from autoptsclient_common import get_unique_name
+
+
+iut_device_name = 'Tester'.encode()
```

Comment:
I think this should be removed in favor of new get_unique_name(), no?

Reply:
Yes, these indeed can be remove. I also found a bug in gap.py with calculating name length: it was in decimal format, not hex. Then, when PTS was checking packet payload it was discarding it, since 10 is not 0a. I included this in this commit, since it only occurs with longer names, when ad_str_name_short_len>=10. I also removed unused code that relied on iut_device_name being outside `set_pixits()`